cron/k8s/worker.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: scorecard-batch-worker
+spec:
+  replicas: 5
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: worker
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: worker
+    spec:
+      containers:
+      - name: worker
+        image: gcr.io/openssf/scorecard-batch-worker
+        imagePullPolicy: Always
+        env:
+        - name: GITHUB_AUTH_TOKEN
```

Comment:
The question I have about the token is that. In the existing single binary without scaling horizontally, the round-robin token https://github.com/ossf/scorecard/blob/587f41117bc9588bf940330195331d5049312c96/roundtripper/github.go#L62 will pick the next one based on the `counter`.

How is this going to work with horizontal scaling? 

If the 5 of them start and everything's going to use the first token. Have we tested this model and seen how are hitting the limits in the GitHub API? 

Ideally, we would need to test the sample set with enough size and a new set of tokens to compare the existing one with this.

Reply:
Good question. This is a concern that did come up in our design discussions. TLDR is that for the sake of simplicity of the design, we chose to start out with this, where multiple jobs share tokens (although it might cause us some issues with rate limits). We can look into improving this once we have a more stable framework.